packages/peregrine/lib/context/__tests__/__snapshots__/header.spec.js.snap

```diff
@@ -0,0 +1,22 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`provides state and actions via context 1`] = `
+Array [
+  Array [
+    Object {
+      "headerRef": Object {
+        "current": null,
```

Comment:
Expected because we have assigned the ref to a temp component that is not mounted.